docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: .docker/app/Dockerfile
+    stdin_open: true
```

Comment:
Why?

Reply:
Without this you wont be able to do a `binding.pry`  that connects to terminal output (it just wont respond to your keys)

Reply:
Here is some better reference:  http://www.chris-kelly.net/2016/07/25/debugging-rails-with-pry-within-a-docker-container/